frontend/pages/index.html

```diff
@@ -1,19 +1,23 @@
-<!DOCTYPE html>
-<html lang="en">
-	<head>
-		<meta charset="UTF-8">
-		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-		<link rel="stylesheet" href="../styles/index.css">
-		<script type="module" src="../scripts/index.js"></script> 
-		<title>Tetris</title>
-	</head>
-
-<body>
-	<span>Hello World! We're going to make Tetris. </span>
-	<button class="a" onclick="window.location.href = 'game.html';">play</button>
-	<button class="a" id="credits" onclick="window.location.href = 'credits.html';">credits</button>
-  <button class="a" onclick="window.location.href = 'howToPlay.html';">How To Play</button>
-</body>
-
-</html>
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="../styles/index.css">
+		<script type="module" src="../scripts/index.js"></script> 
+		<title>Tetris</title>
+	</head>
+
+<body>
+	<div class="allElements"> 
+		<span class = "textElements">Hello Softwire! We have made Tetris! </span>
+		<br>
+		<button class="buttonElements" onclick="window.location.href = 'game.html';">Play Tetris!</button>
+		<br>
+		<button class="buttonElements" id="credits" onclick="window.location.href = 'credits.html';">Credits</button>
+	    <button class="buttonElements" onclick="window.location.href = 'howToPlay.html';">How To Play</button>
```

Comment:
Have a look at how classes are defined elsewhere (e.g. on game.html) - we want to keep styling consistent and naming clear! Can you be more detailed with how you're naming them?